styles/components/contact.css

```diff
@@ -0,0 +1,42 @@
+.contact {
+  text-align: center;
+  padding: 1.5rem 0;
+  border-top: 1px solid #9bd0ef;
```

Comment:
Maybe you have not the time yet to replace the color with your color variable. Use command + f in the document in vs code and replace it with the variable.